admin-dev/themes/new-theme/js/pages/product/components/dropzone/DropzoneWindow.vue

```diff
@@ -29,7 +29,7 @@
         <p
           class="dropzone-window-number"
           v-html="
-            $t('window.selectedFiles', { '%filesNb%': selectedFiles.length })
+            $t('window.selectedFiles', { 'filesNb': selectedFiles.length })
```

Comment:
I don't understand why in some cases you modified the placeholder by `{filesNb}` (using the `{}` delimiters) and in some other cases you simply use the placeholder name `filesNb` Are the two notations equivalent? If they are why not use the same convention all the time?

(this question is beyond the one about the wording changes)